les/client.go

```diff
@@ -115,7 +115,7 @@ func New(stack *node.Node, config *ethconfig.Config) (*LightEthereum, error) {
 		reqDist:         newRequestDistributor(peers, &mclock.System{}),
 		accountManager:  stack.AccountManager(),
 		merger:          merger,
-		engine:          ethconfig.CreateConsensusEngine(stack, chainConfig, config, nil, false, chainDb),
+		engine:          ethconfig.CreateConsensusEngine(stack, chainConfig, config, nil, false, false, chainDb),
```

Comment:
need to pass in `cfg.PrivateKeyForDataFieldSignature` to `CreateConsensusEngine()` instead of `false`

Reply:
This is a redundant parameter pass based on an old approach, so removing entirely, thanks